src/lib/react-query.ts

```diff
@@ -1,9 +1,10 @@
-import { QueryClient, DefaultOptions } from "react-query"
+import { QueryClient } from "react-query"
+export { QueryClientProvider } from "react-query"
```

Comment:
Exports both the provider and its default config now. Look for a PR in Gwen that does this:

```js
import { QueryClientProvider, queryClient } from "@liftedinit/ui";
...
return (
  <QueryClientProvider client={queryClient}>
  ...
  </QueryClientProvider>
);
```